README.md

```diff
@@ -72,6 +132,30 @@ Explore our detailed CLV examples using data from the [`lifetimes`](https://gith
 | **Continuous** | Buying groceries    | Audible                         |
 | **Discrete**   | Cinema ticket       | Monthly or yearly subscriptions |
 
+### CLV Quickstart
+
+```python
+import matplotlib.pyplot as plt
+import pandas as pd
+import seaborn as sns
+from pymc_marketing import clv
+
+
+data_url = "https://raw.githubusercontent.com/pymc-labs/pymc-marketing/main/data/clv_quickstart.csv"
+data = pd.read_csv(data_url)
+data['customer_id'] = data.index
+
+beta_geo_model = clv.BetaGeoModel(data=data)
+
+beta_geo_model.fit()
+```
+
+Once fitted, we can use the model to predict the number of future purchases for known customers, the probability that they are still alive, and get various visualizations plotted.
+
+![](/docs/source/_static/expected_purchases.png)
+
+See the Examples section for more on this.
+
 ----
```

Comment:
Break not need imo with the section below

Reply:
ok! Fixed in https://github.com/pymc-labs/pymc-marketing/pull/653/commits/489510c17611fb4424ec759a980d1f21fbff1e8d